es6.md

```diff
@@ -75,9 +82,16 @@ A `yield` expression.
 ## ObjectPattern
 
 ```js
+interface PatternProperty <: Property {
+    type: "Property"; // inherited
+    value: Pattern;
+    kind: "init";
+    method: false;
```

Comment:
@impinball This is something that is already present in implementations; since this is a living standard, nothing stops us from changing docs when implementations will be changed too.